gradle/verification-metadata.xml

```diff
@@ -5849,6 +5849,14 @@
             <sha256 value="8e93f379f1bd40d793367636ca501ce969c7ee08e5595a6e0249973041388b9b" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.yaml" name="snakeyaml" version="2.0">
+         <artifact name="snakeyaml-2.0.jar">
+            <sha256 value="880c9d896e4b74a06c549c15ca496450165d6909fa15d7e662bee8f6a66d7afa" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="snakeyaml-2.0.pom">
+            <sha256 value="43c761f92b549c8b08fb992080253e49f0a983e544ef0663c1f4f9d68eb52616" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
```

Comment:
Can older versions of snake yaml be removed from here now? Not sure what the process is normally...thought it was automated though 🤷